Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -85,6 +86,26 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         return viewController
     }
 
+    static func createWithTemplate(
+        client: GithubClient,
+        owner: String,
+        repo: String,
+        template: String,
+        signature: IssueSignatureType? = nil) -> NewIssueTableViewController? {
+
+        let storyboard = UIStoryboard(name: "NewIssue", bundle: nil)
+
+        let viewController = storyboard.instantiateInitialViewController() as? NewIssueTableViewController
```

Comment:
Should we exit scope if we can't do this? Can't imagine that that would be the case unless somebody removes the storyboard or changes its name 😛

Reply:
Would be awesome! Can also do this in a follow up sometime.